apps/solhint/src/profile.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "solhint",
+  "displayName": "Solhint Linter",
+  "description": "Linter for Solidity",
+  "version": "0.1.0",
+  "events": ["lintOnCompilationFinished", "lintingFinished"],
+  "methods": [
+    "lintWithoutCompilationCustomAction",
+    "lintOnCompilation",
+    "lintContractCustomAction"
+  ],
+  "kind": "none",
+  "icon": "https://raw.githubusercontent.com/protofire/solhint/master/solhint-icon.png",  
+  "location": "sidePanel",
```

Comment:
if you don't want UI you can set this to hiddenPanel

Reply:
true. Mistaken here.